README.md

```diff
@@ -11,18 +11,18 @@ The NativeScript CLI lets you create, build, and deploy NativeScript-based proje
 * [Supported Platforms](#supported-platforms "The mobile platforms you can target with NativeScript")
 * [System Requirements](#system-requirements "The hardware and software requirements for setting up and working with the NativeScript CLI")
 * [Installation](#installation "How to configure and install the NativeScript CLI")
-	* [Install the NativeScript CLI](#install-the-nativescript-cli)
-	* [Configure Proxy Usage](#configure-proxy-usage)
+    * [Install the NativeScript CLI](#install-the-nativescript-cli)
```

Comment:
well, CLI uses tabs 😺 
However this is not a big deal, it's just the README and I believe we'll have to use the same guidelines in all products (we'll have to discuss this further).